### PR TITLE
COMMERCE-12277 Tier Pricing Table with UOM

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/COMMERCE-12277

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — COMMENTED** [review]
Why are we handling the row click? And what it's is doing that function?

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/tier_price/TierPrice.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+					price: tierPrice.priceFormatted,
+					quantity: tierPrice.quantity,
+				});
+			}
+
+			setRows(rows);
+		}
+	};
+
+	const handleRowClick = (tierPrice) => {
```
[the remark this answers is not in the document]
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/tier_price/TierPrice.js:133]: Answered offline.. Please remove it, we will add it back when it's actually needed

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Please remove the unused function/event, let's add it when we have more descriptive requirements

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/tier_price/TierPrice.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (smottal) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/tier_price/TierPrice.js:133]: It's used to generate an event when the row is clicked. At the moment this event is not listened to by anything (there is no information about it in the history) but in the previous version of the table the quantity selector was updated on the selection of a price level this component is written with this logic

**Comment by smottal** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by smottal** [comment]
ci:test:stable

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
The failures are unrelated to the PR it can be forwarded. Thanks!

@victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**